src/app/modules/workspace/workspace.controller.ts

```diff
@@ -15,7 +15,7 @@ const logger = createLoggerWithLabel(module)
 // Validators
 const workspaceTitleValidator = celebrate({
   [Segments.BODY]: {
-    title: Joi.string().min(4).max(50).required(),
+    title: Joi.string().min(4).max(25).required(),
```

Comment:
same here - if we can reuse the constants in `workspaceValidation`

Reply:
Yes thats a lot cleaner thanks shuli!

[f9c3c65](https://github.com/opengovsg/FormSG/pull/6836/commits/f9c3c6501b6c1b737a0a84c2f406ca840823961f)